openwisp_monitoring/monitoring/configuration.py

```diff
@@ -224,6 +224,27 @@ def _get_access_tech():
             }
         },
     },
+    'general_clients': {
+        'label': _('Clients'),
+        'name': '{name}',
+        'key': 'cliwifi_clientsents',
+        'field_name': 'clients',
+        'charts': {
+            'gen_wifi_clients': {
+                'type': 'bar',
+                'label': _('WiFi clients'),
+                'title': _('WiFi clients'),
+                'description': _('WiFi clients associated to the wireless interface.'),
+                'summary_labels': [_('Total Unique WiFi clients')],
+                'unit': '',
+                'order': 230,
+                'query': chart_query['general_wifi_clients'],
+                'query_default_param': {
+                    'organization_lookup': '',
+                },
```

Comment:
The `Chart.clean` method validates timeseries DB query. While creating Chart in database migration, the organization_lookup is absent. Therefore, I have added default query params.